molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/settings/base.py

```diff
@@ -64,11 +64,15 @@
 
     'molo.core',
     '{{cookiecutter.app_name}}',
+    'mptt',
+    'django.contrib.sites',
```

Comment:
Please move `django.contrib.sites` to the top block (line 47) just after `staticfiles`